src/components/sortmenu/sortmenu.js

```diff
@@ -90,7 +95,7 @@ define(['require', 'dom', 'focusManager', 'dialogHelper', 'loading', 'layoutMana
                     }
 
                     if (submitted) {
-                        saveValues(dlg, options.settings, options.settingsKey);
+                        saveValues(dlg, options.settingsKey);
```

Comment:
why options.settings was removed?

Reply:
`options.settings` was unused in `saveValues`.